index.bs

```diff
@@ -3435,6 +3441,150 @@ The [=remote end steps=] with <var ignore>session</var> and |command parameters|
 
 </div>
 
+#### The browsingContext.print Command ####  {#command-browsingContext-print}
+
+The <dfn export for=commands>browsingContext.print</dfn> command
+creates a paginated representation of a document, and returns it as a
+PDF document represented as a Base64-encoded string.
+
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+      <pre class="cddl remote-cddl">
+      browsingContext.Print = {
+        method: "browsingContext.print",
+        params: browsingContext.PrintParameters
+      }
+
+      browsingContext.PrintParameters = {
+        context: browsingContext.BrowsingContext,
+        ?background: bool .default false,
+        ?margin: browsingContext.PrintMarginParameters,
+        ?orientation: ("portrait" / "landscape") .default "portrait",
+        ?page: browsingContext.PrintPageParameters,
+        ?pageRanges: [*text],
+        ?scale: 0.1..2.0 .default 1,
```

Comment:
Would this require a float type as well?

Reply:
@jgraham I actually meant more the following: `?scale: (int 0.1..2.0) .default 1.0,` or do we want to force `float` type only?

Reply:
There isn't a difference in JS/JSON between number types; if we allow floats we also allow integers i.e. there's no semantic difference in JSON between `1` and `1.0`. The integer check is basically post-processing of the form "does this JS number represent an exact integer".

Reply:
Ok, so we can actually omit specifying the type when a default value or a range is given. So I'm fine with what we have here.